tools/package_executable.py

```diff
@@ -0,0 +1,129 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--temp_dir", required=True)
+    parser.add_argument("--conda_env", required=True)
+    parser.add_argument("--conda_path")
+    parser.add_argument("--config_mode", default="Shipping")
+    parser.add_argument("--unreal_engine_dir", required=True)
+    parser.add_argument("--num_parallel_jobs", type=int, default=1)
+    parser.add_argument("--target_platform", required=True, help="Values accepted are Win64, Linux, Mac")
+    parser.add_argument("--output_dir", default=os.path.dirname(os.path.realpath(__file__)))
+    parser.add_argument("--output_tag", required=True)
+    args = parser.parse_args()
+
+    repo_dir = os.path.join(os.path.realpath(args.temp_dir), "spear")
+
+    # need conda_path on unix systems
+    if sys.platform in [ "darwin", "linux"]:
+        assert args.conda_path
+
+    # remove any contents already present for a clean build
+    if os.path.exists(repo_dir):
+        shutil.rmtree(repo_dir)
+        os.makedirs(repo_dir)
+
+    # clone repo along with submodules  
+    cmd = ["git", "clone", "--recurse-submodules", "https://github.com/isl-org/spear", repo_dir]
+    print(f"[SPEAR | build.py] Executing: {' '.join(cmd)}")
+    cmd_result = subprocess.run(cmd)
+    assert cmd_result.returncode == 0
+
+    # change working directory
+    cwd = os.getcwd()
+    print(f"[SPEAR | build.py] Changing working directory to {os.path.join(repo_dir, 'tools')}")
+    os.chdir(os.path.join(repo_dir, "tools"))
+
+    # build thirdparty libs
+    if sys.platform == "win32":
+        cmd = ["python", "build_third_party_libs.py", "--num_parallel_jobs", f"{args.num_parallel_jobs}"]
```

Comment:
Don't do this in a platform-specific way.

Reply:
updated